Signal/src/contact/SystemContactsFetcher.swift

```diff
@@ -145,3 +186,22 @@ class SystemContactsFetcher: NSObject {
     }
 
 }
+
+struct HashableArray<Element: Hashable>: Hashable {
```

Comment:
Why a collection?  Why not just a function that computes the hash for a collection of Hashables?